modules/identity/src/Volo.Abp.Identity.Web/Pages/Identity/Roles/EditModal.cshtml

```diff
@@ -55,7 +56,7 @@
                                    asp-for="Role.ExtraProperties[propertyInfo.Name]"
                                    label="@propertyInfo.GetLocalizedDisplayName(StringLocalizerFactory)"
                                    asp-format="@propertyInfo.GetInputFormatOrNull()"
-                                   value="@propertyInfo.GetInputValueOrNull(Model.Role.ExtraProperties[propertyInfo.Name])" />
```

Comment:
This change may break something. Look at the implementation of the `GetInputValueOrNull` method:

https://github.com/abpframework/abp/blob/3fcdeeeb05810b7a2e162f212ef435b81d412aa5/framework/src/Volo.Abp.AspNetCore.Mvc.UI/Volo/Abp/ObjectExtending/MvcUiObjectExtensionPropertyInfoExtensions.cs#L53-L67

`GetProperty` is doing a completely different thing.

Reply:
Done.